ios/VisionCameraPluginInatVision/VisionCameraPluginInatVision.m

```diff
@@ -174,8 +174,20 @@ - (id)callback:(Frame*)frame withArguments:(NSDictionary*)arguments {
         results = visionScores;
     }
 
+    // Get a pointer to the raw data
+    float *dataPointer = (float *)results.dataPointer;
+    NSUInteger count = results.count;
+    // Use vDSP to find the maximum value
+    float topCombinedScore;
+    vDSP_maxv(dataPointer, 1, &topCombinedScore, count);
+    // define some cutoff based on a percentage of the top combined score. Taxa with
+    // scores below the cutoff will be ignored when aggregating scores up the taxonomy
+    float scoreRatioCutoff = 0.001;
+    float cutoff = topCombinedScore * scoreRatioCutoff;
+
     // Setup taxonomy
     VCPTaxonomy *taxonomy = [VisionCameraPluginInatVisionPlugin taxonomyWithTaxonomyFile:taxonomyPath];
+    [taxonomy setTaxonomyRollupCutoff:cutoff];
```

Comment:
The API decision I made here was that by default the cutoff of topCombinedScore * 0.001 is always on but you can override with any number you choose.

Reply:
makes sense